packages/typography/src/utils/createStyles.js

```diff
@@ -28,6 +28,7 @@ const setStyles = (styles: Object = {}, els: string | string[], rules: Object) =
 
 module.exports = (vr: any, options: OptionsType) => {
   let styles = {}
+  let { fontSize, lineHeight } = vr.rhythm(1)
```

Comment:
You should use `options.baseFontSize` and `options.baseLineHeight`. The rhythm function just returns an rem value.